packages/infra/lib/api-stack.ts

```diff
@@ -595,6 +595,21 @@ export class APIStack extends Stack {
       sentryDsn: props.config.lambdasSentryDSN,
     });
 
+    this.setupBulkUrlSigningLambda({
+      lambdaLayers,
+      vpc: this.vpc,
+      medicalDocumentsBucket: medicalDocumentsBucket,
+      fhirServerUrl: props.config.fhirServerUrl,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: slackNotification?.alarmAction,
+      searchEndpoint: ccdaSearchDomain.domainEndpoint,
+      searchIndex: ccdaSearchIndexName,
+      searchUserName: ccdaSearchUserName,
+      searchPassword: ccdaSearchSecret.secretValue.unsafeUnwrap(),
```

Comment:
We should not pass secrets as env vars. See `unsafe` on the function name.

Look how we do this for the exact same secret on `sqs-to-opensearch-xml`.